methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+def bottles_of_beer(bottles)
+  line_1 = "#{bottles} bottles of beer on the wall, #{bottles} bottles of beer.
```

Comment:
Could you create a method that pluralizes a string?  `pluralize('bottle', number)` should return the string `"#{number} #{pluralized_string}"`